test/GCHP/README.md

```diff
@@ -0,0 +1,68 @@
+# README for Integration Tests
+
+## Overview:
+
+This directory contains scripts to submit automated GCHP integration tests, which will test the following process:
+
+1. GCClassic run directory creation
+2. GCClassic code configuration and compilation
+3. GCClassic execution
+
+Integration tests are mostly intended to be run by GEOS-Chem developers, rather than by end users.
+
+## Files
+
+- `intTest.sh`
+  - Driver script to run GCHP integration tests
+- `intTestCreate.sh`
+  - Script to create GCHP run directories for integration tests
+- `intTestCompile.sh`
+  - Script to compile GCHP Classic executables
+- `intTestExecute.sh`
+   - Script to run GCHP Classic simulations
```

Comment:
Remove "Classic" from these two lines

Reply:
Fixed in commit 79b6cb5643279e709052a00eb9a82f18d6f47019